src/main/resources/handlebars/JavaSpring/libraries/spring-boot/homeController.mustache

```diff
@@ -10,7 +10,13 @@ import org.springframework.web.bind.annotation.RequestMapping;
 public class HomeController {
     @RequestMapping(value = "/")
     public String index() {
+        {{#useOas2}}
         System.out.println("swagger-ui.html");
         return "redirect:swagger-ui.html";
+        {{/useOas2}}
+        {{^useOas2}}
+        System.out.println("/swagger-ui/index.html");
```

Comment:
The Swagger UI packaging is different for Springfox 2.x / 3.x release lines, accomodating this change

Reply:
is the print statement required?

Reply:
Not at all, it was there so I just preserved it for OAS 3.x as well, could be gone completely :)